force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -124,13 +173,32 @@ public with sharing class DummyJSONCallout {
 	@TestVisible // Allows test class to see this method. Since it is private, it would not be visible otherwise.
 	private static String generateDummyJsonUserPayload(String contactId) {
 		// Query the contact to get the field values to generate the JSON payload.
+		List<Contact> selectedContact = new List<Contact>();
+
+		// if (Contact.SObjectType.getDescribe().isAccessible() && Schema.SObjectType.Contact.fields.FirstName.isAccessible() &&
+        //     Schema.SObjectType.Contact.fields.Phone.isAccessible() && Schema.SObjectType.Contact.fields.LastName.isAccessible() &&
+        //     Schema.SObjectType.Contact.fields.Email.isAccessible()) {
+			selectedContact = [SELECT Id, FirstName, LastName, Email, Phone FROM Contact WHERE Id = :contactId];
+		//}
 
 		// Create a map of the field values.
+		Map<String, Object> jsonMap = new Map<String, Object>();
+		
+		// if a contact is selected, set the values to the jsonMap object.
+		if (selectedContact.size() > 0) {
```

Comment:
Nice work including stand-in values for nulls so you know values are not mapped because they do not exist, not because of an error of some kind